### PR TITLE
Allow URLs to include accented characters

### PR DESCRIPTION
**Pull request by edwinjoseph**
## Description
When sanitizing url with accents they get omitted from the return value. Ensure that urls with accents are allowed.

Fixes issue #20 

## Acceptance Criteria
- [ ] URLs with accents are allowed

### CODE REVIEW

**Comment by crookedneighbor** [comment]
Actually, I think this is missing some accent characters. Let me offer some PR suggestions.

File: index.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 var invalidPrototcolRegex = /^(%20|\s)*(javascript|data)/im;
-var ctrlCharactersRegex = /[^\x20-\x7E]/gmi;
+var ctrlCharactersRegex = /[^\x20-\x7EÀ-ÖØ-öø-ÿ]/gmi;
```
Comment (crookedneighbor) [inline index.js]:
```suggestion
var ctrlCharactersRegex = /[^\x20-\x7EÀ-ž]/gmi;
```

File: test/test.js [hunk 2]
```diff
@@ -88,6 +88,10 @@ describe('sanitizeUrl', function () {
     expect(sanitizeUrl('mailto:test@example.com?subject=hello+world')).to.equal('mailto:test@example.com?subject=hello+world');
   });
 
+  it('does not alter urls with accented characters', function () {
+    expect(sanitizeUrl('www.example.com/with-áccents')).to.equal('www.example.com/with-áccents');
```
Comment (crookedneighbor) [inline test/test.js]:
```suggestion
    expect(sanitizeUrl('www.example.com/with-áccêntš')).to.equal('www.example.com/with-áccêntš');
```

**Comment by crookedneighbor** [comment]
@edwinjoseph ping

**Comment by edwinjoseph** [comment]
> @edwinjoseph ping

Yup sorry, completely forgot. I've updated this and ran the tests, all good from my end.

**Comment by crookedneighbor** [comment]
This will go out with the next release.

**Pull request merged**